modules/nlu/src/backend/prediction-handler.test.ts

```diff
@@ -1,6 +1,7 @@
 import '../../../../src/bp/sdk/botpress.d'
+import * as nluCore from '../../../../src/bp/nlu-core'
```

Comment:
This import is not correct. Modules should never have to require source code from the main code

Reply:
this is just a test file